### DISCUSSION TITLE
UE4 Games Titles from WP, List A-C

### DISCUSSION THREAD

**Discussion by Robert-W-K** [TheNewJavaman/unreal-vr#1] [General]
Explanations:

* **Title**: Name
* **Point of View**: How the camera in the game follows the player or is the player
* **Button Mapping**: the game might not need special motion tracked weapons or hands; static actions could be completed with a button
* **Head Tracking**: the game would be impoved with VR if it were head tracked to view the world
* **Motion Tracked Hands/Weapons**: there are weapons that could be tracked with the hand controllers, or other hand actions
* **Flashlight**: there seems to be a separate flashlight in the game
* **Shooting**: there is a gun action in the game or something that shoots a projectile and needs a response to doing so (vibration, other)
* **Interaction**: there might be some object or trigger interaction in this game that would have to be accounted for in the control scheme, but it might not necessarily be manipulated with a physical hand: think of picking up things in games with "phantom" hands
* **Needs UI?**: this game has a lot of UI elements that can't be reproduced through gameplay. Think Real Time Strategy/Games with lots of menus, or even perhaps would need a GUI for a vehicle
* **Other notes**: Things that I observed in the game that might be important for VR conversion

For reference, I have not played all of these games and might be wrong about some elements. Feel free to let me know if there need to be any changes, or if I missed games on the list. So far I've reviewed about 1/6th of the content from the list.

| # |Game Title                                 |Point of View                         |Button Mapping                  |Head Tracking|Motion Tracked Hands/Weapons|Flashlight|Shooting|Interaction|Needs UI?|Other notes:                                   |
|---|-------------------------------------------|--------------------------------------|:------------------------------:|:-----------:|:--------------------------:|:--------:|:------:|:---------:|:-------:|-----------------------------------------------|
|1  |9 Monkeys of Shaolin                       |Side-scrolling                        |               x                |             |                            |          |        |           |         |                                               |
|2  |404Sight                                   |3rd-Person rear view                  |               x                |       x      |                            |          |        |           |         |                                               |
|3  |A Way Out                                  |3rd-Person multi view                 |               x                |             |                            |          |        |           |         |                                               |
|4  |Absolver                                   |3rd-Person multi view                 |               x                |             |                            |          |        |           |         |                                               |
|5  |Abzû                                       |3rd-Person rear view                  |               x                |       x      |                            |          |        |           |         |                                               |
|6  |Acacia: Time Trails                        |3rd-Person rear view                  |               x                |             |                            |          |        |           |         |                                               |
|7  |Ace Combat 7: Skies Unknown                |Flight Sim—different views            |               x                |      x      |                            |          |        |           |         |                                               |
|8  |Ad Infinitum                               |1st-Person view                       |               x                |       x      |              x              |    x     |    x    |     x      |         |                                               |
|9  |Adrift                                     |1st-Person view                       |               x                |      x      |             x              |          |        |           |         |                                               |
|10 |Age of Blood                               |3rd-Person rear view                  |               x                |      x      |                            |          |        |           |         |                                               |
|11 |AION Next-Gen                              |3rd-person rear view                  |               x                |      x      |                            |          |        |           |     x    |Role Playing Game (like WoW)                   |
|12 |AIPD                                       |Top Down Shooter                      |               x                |             |                            |          |        |           |         |Bullet Hell-style                              |
|13 |A.L.I.C.E.                                 |Adventure                             |                                |             |                            |          |        |           |         |                                               |
|14 |Aliens: Fireteam Elite                     |3rd-Person over-the-shoulder rear view|               x                |      x      |             x              |          |   x    |     x     |         |                                               |
|15 |Allison Road                               |Survival horror                       |                                |             |                            |          |        |           |         |                                               |
|16 |Alone in the Dark: Illumination            |3rd-Person over-the-shoulder rear view|                                |      x      |                            |          |   x    |     x     |         |Like Resident Evil                             |
|17 |Amid Evil                                  |1st-Person view                       |Being worked on by the team—skip|             |                            |          |        |           |         |                                               |
|18 |Ancestors: The Humankind Odyssey           |3rd-Person multi view                 |                                |      x      |             x              |          |        |           |         |                                               |
|19 |Apsulov: End of Gods                       |1st-Person view                       |                                |      x      |             x              |          |        |     x     |         |                                               |
|20 |Aquanox Deep Descent                       |Flight Sim—1st person view            |               x                |      x      |                            |          |        |           |         |Like Descent                                   |
|21 |Ark: Survival Evolved                      |1st-Person/3rd-Person                 |                                |      x      |             x              |          |        |           |         |Like Green Hell VR sort of                     |
|22 |Ashen                                      |3rd-Person rear view                  |               x                |      x      |                            |          |        |           |         |Could involve hands/interaction/weapons        |
|23 |Assetto Corsa Competizione                 |Racing simulation                     |  Already natively implemented  |             |                            |          |        |           |         |Not sure how it works                          |
|24 |Astray                                     |1st-Person view                       |                                |      x      |                            |    x     |        |     x     |         |                                               |
|25 |Astroneer                                  |3rd-Person                            |                                |      x      |             x              |          |        |     x     |         |                                               |
|26 |Atomic Heart                               |1st-Person view                       |                                |      x      |             x              |          |        |     x     |         |                                               |
|27 |Automation                                 |...simulation of some kind            |                                |      x      |                            |          |        |           |    x    |                                               |
|28 |Aven Colony                                |3rd-Person Top Down Real Time Strategy|                                |      x      |             x              |          |        |           |    x    |Would be interesting real-time-strategy VR game|
|29 |Back 4 Blood                               |1st-Person view                       |                                |      x      |             x              |          |   x    |     x     |         |Could be better done than Vorpx                |
|30 |Bacon Man: An Adventure                    |3rd-Person platformer                 |               x                |      x      |                            |          |        |           |         |                                               |
|31 |Balan Wonderworld                          |3rd-Person multi view                 |               x                |      x      |                            |          |        |     x     |         |                                               |
|32 |Barrage                                    |RTS, First-person shooter             |                                |             |                            |          |        |           |         |                                               |
|33 |Batman: Arkham VR                          |Adventure                             |       Already a VR Game        |             |                            |          |        |           |         |                                               |
|34 |Battalion 1944                             |1st-Person view                       |                                |      x      |             x              |          |   x    |     x     |         |                                               |
|35 |Battlefleet Gothic: Armada                 |3rd-Person Top Down Real Time Strategy|                                |      x      |             x              |          |        |           |    x    |                                               |
|36 |Beyond Flesh and Blood                     |3rd-Person multi view                 |                                |      x      |             x              |          |   x    |     x     |         |Mech style game                                |
|37 |Biomutant                                  |3rd-Person multi view                 |               x                |      x      |                            |          |        |           |         |                                               |
|38 |BIOS                                       |1st-Person view                       |                                |      x      |             x              |          |   x    |     x     |         |                                               |
|39 |Blade & Soul                               |3rd-Person rear view                  |               x                |      x      |                            |          |        |           |         |                                               |
|40 |Blair Witch                                |Survival Horror                       |       Already a VR Game        |             |                            |          |        |           |         |                                               |
|41 |Bleeding Edge                              |3rd-Person multi view                 |               x                 |      x      |                            |          |        |           |         |Skateboard/Vehicle                             |
|42 |Bless Unleashed                            |3rd-Person rear view                  |               x                |      x      |                            |          |        |           |         |                                               |
|43 |Bloodstained: Ritual of the Night          |3rd-Person platformer                 |               x                |      x      |                            |          |        |           |         |                                               |
|44 |Blue Protocol                              |3rd-person multi view                 |                                |             |                            |          |        |           |         |                                               |
|45 |Borderlands 3                              |1st-person view                       |                                |      x      |             x              |          |   x    |     x     |         |                                               |
|46 |Bravely Default II                         |RPG                                   |              ???               |             |                            |          |        |           |         |                                               |
|47 |Bright Memory                              |1st-person view                       |                                |      x      |             x              |          |   x    |     x     |         |                                               |
|48 |Bus Simulator 18                           |1st-person view simulation            |                                |      x      |             x              |          |        |           |    x    |Vehicles                                       |
|49 |Bus Simulator 21                           |1st-person view simulation            |                                |      x      |             x              |          |        |           |    x    |Vehicles                                       |
|50 |Caffeine                                   |1st-person view                       |                                |      x      |                            |    x     |        |     x     |         |                                               |
|51 |Call of Cthulhu                            |1st-person view                       |                                |      x      |                            |          |        |     x     |    x    |                                               |
|52 |Call of the Sea                            |1st-person view                       |                                |      x      |                            |          |        |     x     |    x    |                                               |
|53 |Century: Age of Ashes                      |3rd-Person rear view                  |                                |      x      |                            |          |   x    |           |         |Flying                                         |
|54 |Chivalry 2                                 |3rd-person multi view                 |                                |      x      |             x              |          |        |     x     |         |                                               |
|55 |City of Titans                             |MMORPG                                |                                |             |                            |          |        |           |         |                                               |
|56 |Code Vein                                  |3rd-person multi view                 |               x                |      x      |                            |          |        |           |         |                                               |
|57 |Colony Ship: A Post-Earth Role Playing Game|3rd-Person Top Down Real Time Strategy|               x                |      x      |                            |          |        |           |    x    |                                               |
|58 |Conan Exiles                               |1st-person/3rd-Person                 |                                |      x      |             x              |          |        |     x     |    x    |                                               |
|59 |Conarium                                   |1st-person                            |                                |      x      |                            |    x     |        |     x     |         |                                               |
|60 |Consortium: The Tower Prophecy             |1st-person                            |                                |      x      |             x              |          |        |           |         |Flying                                         |
|61 |Courage                                    |Puzzle                                |                                |             |                            |          |        |           |         |                                               |
|62 |Crackdown 3                                |3rd-Person rear view                  |                                |      x      |             x              |          |        |     x     |    x    |Vehicles                                       |
|63 |Crash Bandicoot 4: It's About Time         |3rd-Person platformer                 |               x                |      x      |                            |          |        |           |         |                                               |
|64 |Crash Force                                |3rd-Person rear view                  |                                |      x      |                            |          |   x    |           |    x    |                                               |

**Comment by TheNewJavaman** [comment]
Oh wow, I really want to play Astroneer again, but in VR :)

**Reply by Robert-W-K** [reply]
Maybe you will!
